internal/template-bundle/bundle_test.go

```diff
@@ -1,48 +1,61 @@
 package template_bundle
 
 import (
+	"runtime"
 	"testing"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
+func retrieveSuffix() string {
+	switch runtime.GOARCH {
+	case "amd64":
+		return ""
+	case "s390x":
+		return "-s390x"
+	default:
+		return ""
+	}
+}
+
 var _ = Describe("Template bundle", Ordered, func() {
 	var (
 		testBundle Bundle
+		suffix     string
```

Comment:
nit - Please give it a more descriptive name, like `nameSuffix`.

Reply:
Done.